static/js/main.js

```diff
@@ -134,4 +134,42 @@ $(() => {
 
   // Suppress form submission:
   $("form.identities").submit(() => false);
+
+  // Allow follow and unfollow:
+  $(".follow-button").click(evt => {
```

Comment:
I would be tempted to do this as two distinct buttons, which are shown or hidden based on follow status, rather than as one omnibutton with an if. Something to revisit in a UI redevelopment, though, rather than an urgent issue.